packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -503,22 +513,23 @@ export function messageChannelMiddleware(
           case ActionTypes.HANDLE_POINTER_MOVE:
           case ActionTypes.ELEMENT_FROM_POINT_CHANGE:
           case ActionTypes.SET_LOCALE:
-            messageChannel.port1.postMessage(action)
+          case ActionTypes.SET_BREAKPOINTS:
```

Comment:
Was this case (SET_BREAKPOINTS) just missing before?

Reply:
I now see that it's handling the `messageChannel.port1.postMessage(setBreakpoints(breakpoints))` that was removed above 👍🏻